### ISSUE TITLE
Change Video Tutorial URL

### ISSUE THREAD

**Issue by ezys** [SkyTruth/CrowdProjects#23]
Replace the placeholder video url in tutorial.html with this video tutorial. 

http://youtu.be/UxjI3YFiEf4

<iframe width="560" height="315" src="https://www.youtube.com/embed/UxjI3YFiEf4" frameborder="0" allowfullscreen></iframe>

**Comment by ezys** [comment]
<iframe width="560" height="315" src="https://www.youtube.com/embed/UxjI3YFiEf4" frameborder="0" allowfullscreen></iframe>

**Comment by geowurster** [comment]
@ezys when creating tasks like this that must be completed before other tasks its good to note that where appropriate.  In this case this task must be completed BEFORE another task so if you look at #24 I included a note in the first line of the task description.

**Comment by geowurster** [comment]
@ezys Its also a good idea to keep the task description up to date, so instead of adding the `iframe` tag in a comment it would be better to update the description with the new content and then @ me in a comment saying it was updated.

**Comment by ezys** [comment]
My understanding is that these were not necessarily dependent. But I am
aware and have noted dependencies in other tickets.

**Comment by Yolandeezy** [comment]
done https://github.com/SkyTruth/pybossa_tools/commit/ab8212326b86d6c60497fad23f1dd79b6268baef
